src/backend/dx11/src/factory.rs

```diff
@@ -630,7 +630,33 @@ impl core::Factory<R> for Factory {
                 &mut vertex_layout)
         };
         if !winerror::SUCCEEDED(hr) {
-            error!("Failed to create input layout from ?, error {:x}"/*, layouts*/, hr);
+            struct DebugInputLayout<'a> {
```

Comment:
I can remove this if it's unsightly.

I'm working on https://github.com/retep998/winapi-rs/pull/760, which would additionally make this unnecessary.